### DISCUSSION TITLE
$perPageViaRelationship with array of pagination options

### DISCUSSION THREAD

**Discussion by goellner** [laravel/nova-issues#6389] [Ideas & Feature Requests] [answered]
It would be awesome if we could change `perPageViaRelationship` to support adding an array. 

Example why this could be very useful:

I have Spaties Sortable Eloquent package and need to set to sort on a relationship and need to see all the models to sort all of them at once. But we don't need to sort every time we open the resource and by default it could show only 50 models.

If we could set `$perPageViaRelationship = [50, 500, 10000]` and could change this via the pagination options when needed, we could solve this without querying 10000 models everytime.

**Comment by rderks88** [comment]
Would find this very useful. Sometimes you just need to get an overview of a relationship, that you can't get with 5. And you also don't want 25 or more to be loaded all the time.

**Comment by crynobone** [comment, marked as answer]
Will be available in 5.2.0

**Reply by crynobone** [reply]
Released
